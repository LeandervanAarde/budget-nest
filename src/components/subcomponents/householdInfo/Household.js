import React from 'react';

const Household = () => {
    return (
        <table>
            <thead>
            <tr>
                <th><h5>#</h5></th>
                <th><h5>Name</h5></th>
                <th><h5>Amount</h5></th>
                <th><h5>Tax bracket</h5></th>
                <th><h5>Tax amount</h5></th>
                <th><h5>Income after tax</h5></th>
            </tr>
            </thead>
            <tbody>
            <tr>
                <td>1</td>
                <td>Leander</td>
                <td>R 12 000.00</td>
                <td>15%</td>
                <td>R1500.00</td>
                <td>R10 500.00</td>
            </tr>
            </tbody>
        </table>
    );
};

export default Household;