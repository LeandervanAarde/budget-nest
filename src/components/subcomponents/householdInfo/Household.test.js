import { render, screen} from "@testing-library/react";
import { BrowserRouter as Router } from 'react-router-dom';
import userEvent from "@testing-library/user-event";
import Household from "./Household"

describe("Test the household component", () =>{
    test("", ()=>{

    })
});