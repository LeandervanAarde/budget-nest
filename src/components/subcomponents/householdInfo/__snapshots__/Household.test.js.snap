// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test the household component matches snap 1`] = `
<tr>
  <td>
     
  </td>
  <td>
    Leander
    Riaan
    Nina
  </td>
  <td>
     R 
    12000
    300000
    144000
  </td>
  <td />
  <td />
  <td />
</tr>
`;

exports[`Test the household component matches snap 2`] = `
<tr>
  <td>
     
  </td>
  <td>
    Leander
    Riaan
    Nina
  </td>
  <td>
     R 
    12000
    300000
    144000
  </td>
  <td />
  <td />
  <td />
</tr>
`;
